### PR TITLE
#fixes 183 Add a document about how to add a PR on readme.md, Improve PR structure

### PR DESCRIPTION
**Pull request by STEdirisuriya**
This pull request is related to issue https://github.com/fossasia/labyrinth/issues/183

The facilities I want to improve are:[Add a document about how to add a PR on readme.md, Improve PR structure]

Screenshots/Links: Deploy/Preview Links: http://rawgit.com/STEdirisuriya/labyrinth/master/index.html

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,13 +1,23 @@
-This pull request is related to issue # <!-- insert the issue number behind the # or delete the line -->
+## Don't delete anything without give instryctions you to delete it and don't change any thing without words those you have to change.
+### When you change [things like this] pleaser remobe brackets after change it.
```
Comment (abishekvashok) [inline .github/PULL_REQUEST_TEMPLATE.md]: typo 'remobe`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,13 +1,23 @@
-This pull request is related to issue # <!-- insert the issue number behind the # or delete the line -->
+## Don't delete anything without give instryctions you to delete it and don't change any thing without words those you have to change.
```
Comment (abishekvashok) [inline .github/PULL_REQUEST_TEMPLATE.md]: typo `instryctions`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
+### When you change [things like this] pleaser remobe brackets after change it.
+
+# This pull request is related to issue, 
+#<!-- If you fully fixed/improved some isuue/s, please insert the issue number/s behind the # or if you are not fixed/improved some isuue/s completly, but only some of issue/s step/s please insert the issue number/s with step number/s behind the #-->
```
Comment (abishekvashok) [inline .github/PULL_REQUEST_TEMPLATE.md]: issues/s should have been issue(s)
stop/s should have been stop(s)

File: README.md [hunk 4]
```diff
@@ -137,6 +137,22 @@ under the `resolve` sub class.)
 - Add the tiles to the labyrinth, so they are reachable. Please create a small portion of the labyrinth with them to make it more exciting. You may get inspiration from other parts of the labyrinth.
 - Create a pull-request and have it merged.
 
+### Add a new Pull request after solving a issue
+
+- Go to the labyrinth [repository](https://github.com/fossasia/labyrinth). 
+- Go to the [issue tab](https://github.com/fossasia/labyrinth/issues) and find a issue that you want to resolve or improve.
+- Make it and push those changes into your repo.
```
Comment (abishekvashok) [inline README.md]: Can you make `make it` much more precise

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -1,13 +1,21 @@
-This pull request is related to issue # <!-- insert the issue number behind the # or delete the line -->
+## Don't delete anything without give instructions for you to delete it and don't change any thing without words those you have to change.
```
Comment (abishekvashok) [inline .github/PULL_REQUEST_TEMPLATE.md]: This should have been,
Dont delete anything without explicit instruction s from a maintainer.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -1,13 +1,21 @@
...
+### When you change [things like this in brackets] please remember to delete brackets after change it.
```
Comment (abishekvashok) [inline .github/PULL_REQUEST_TEMPLATE.md]: After you change it

**Comment by STEdirisuriya** [comment]
don't say me to squash, cause I am using web editor right now. Because in my cloned repo in local not this file,

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -1,13 +1,21 @@
-This pull request is related to issue # <!-- insert the issue number behind the # or delete the line -->
+## Dont delete anything without explicit instruction s from a maintainer.
```
Comment (abishekvashok) [inline .github/PULL_REQUEST_TEMPLATE.md]: Typo again

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -1,13 +1,21 @@
-This pull request is related to issue # <!-- insert the issue number behind the # or delete the line -->
+## Don't delete anything without explicit instructions from a maintainer.
+### When you change [things like this in brackets] please remember to delete brackets after you change it.
```
Comment (niccokunzmann) [inline .github/PULL_REQUEST_TEMPLATE.md]: That is useful. We have a lot of brackets lately ^^

File: README.md [hunk 9]
```diff
@@ -137,6 +137,22 @@ under the `resolve` sub class.)
...
+- Copy the issue number from issue tab.
+- Go to your forked repository.
+- Go to Pull Reqest tab.
+- Click on New pull request.
+- See there all files changes are ready or not(If not not follow next steps, please add your changes before proceed).
+- Select base fork and head fork if it isn't not selected automatically.
+- Click on Create Pull request.
+- Change PR topic and message as instructions on it.
+- Then click on create pull request.
+<b>Now your pull request is ready!</b> See on [Pull requests](https://github.com/fossasia/labyrinth/pulls)
```
Comment (niccokunzmann) [inline README.md]: In markdown, you can use **the star** to make it bold: `**the star**`

**Comment by niccokunzmann** [comment]
Thanks for your pull request!

**Pull request merged**